src/test/java/build/buildfarm/instance/memory/MemoryInstanceTest.java

```diff
@@ -47,29 +45,30 @@
 import build.bazel.remote.execution.v2.ActionResult;
 import com.google.longrunning.Operation;
 import com.google.protobuf.Any;
-import com.google.protobuf.Duration;
+import com.google.protobuf.InvalidProtocolBufferException;
 import com.google.protobuf.util.Durations;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+
```

Comment:
Was this an auto formatting thing? Current style throughout the code is no empty lines during diff, and sort alpha.